app.py

```diff
@@ -14,7 +14,8 @@
 
 from models import Category, Criterion, Link, Score
 from auth import AuthError, requires_auth
-from services import update_or_create_category
+from services import update_or_create_category, update_or_create_criterion
+import strings
```

Comment:
I made a `strings.py` file for us to store some commonly-used error messages! (ex. X not found, X cannot be changed)